ts/src/binance.ts

```diff
@@ -1327,10 +1327,60 @@ export default class binance extends Exchange {
                     'TRC20': 'TRX',
                     'BEP2': 'BNB',
                     'BEP20': 'BSC',
-                    'OMNI': 'OMNI',
                     'EOS': 'EOS',
                     'SPL': 'SOL', // temporarily keep support for SPL (old name)
                     'SOL': 'SOL', // we shouldn't rename SOL
+                    // 'FIAT': 'FIAT_MONEY', // not unified atm
+                    // 'LEVERAGE_TOKEN': 'ETF', // not unified atm
+                    // 'STAKING': 'STAKING', // not unified atm
+                    'ARBONE': 'ARBITRUM',
```

Comment:
isn't it easier to list only the differences? doing sol: sol does not add much

Reply:
i dont think so, it does not hurt to be there, at least users looking at it,, would know that `SOL` network is unified and not possibly ignored/missed. 
also good to be there as an example for other exchanges too